content/learn/beginner/conditional-branching.md

```diff
@@ -97,7 +97,7 @@ In programming, we can make decisions by comparing values to each other. We can
 types. The result of a comparison is a _boolean_ value, which is either `true` or `false`. Normally, comparisons are
 binary operators; however, in custom commands, we use functions to compare values.
 
-{{< callout context="caution" title="Comparing across Types" icon="outline/alert-triangle" >}}
+{{< callout context="caution" title="Warning: Comparing across Types" icon="outline/alert-triangle" >}}
```

Comment:
Same as a previous comment, if we are using Title Case, I think `across` here should be capitalized.